storage/app/public/sitemap.json

```diff
@@ -0,0 +1,64 @@
+{
+    "categories": [
+        {
+            "private": false,
+            "name": "Algemene pagina's",
+            "links": [
+                {
+                    "name": "home",
+                    "link": "/"
+                },
+                {
+                    "name": "trainingen",
+                    "link": "/training"
+                },
+                {
+                    "name": "evenementen",
+                    "link": "/evenement"
+                },
+                {
+                    "name": "aanmelden",
```

Comment:
Kijk ff of je deze wilt gebruiken of niet

Reply:
^